gbpservice/nfp/core/log.py

```diff
@@ -64,118 +63,70 @@ def findCaller(self):
             break
         return rv
 
+logging.setLoggerClass(WrappedLogger)
+
+if '_LI' in globals().keys():
+    globals()['_LI'] = _LI
+if '_LE' in globals().keys():
+    globals()['_LE'] = _LE
```

Comment:
We need _LW corresponding to warning also, 
For warning logs it is going as unhandled exception